_api-reference/index-apis/clone.md

```diff
@@ -58,7 +57,7 @@ timeout | Time | How long to wait for the request to return. Default is `30s`.
 
 ## Request body
 
-The clone index API operation creates a new target index, so you can specify any [index settings]({{site.url}}{{site.baseurl}}/opensearch/rest-api/index-apis/create-index/#index-settings) and [aliases]({{site.url}}{{site.baseurl}}/opensearch/index-alias) to apply to the target index.
+The clone index API operation creates a new target index, so you can specify any [index settings]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#index-settings) and [aliases]({{site.url}}{{site.baseurl}}/opensearch/index-alias) to apply to the target index.
```

Comment:
Capitalize links:

`[Index settings]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#index-settings)`

`[Index aliases]({{site.url}}{{site.baseurl}}/opensearch/index-alias)`

Reply:
actually, how it's used in-line in the sentence, lowercase is correct here.
If it was "To learn more about. .  ., see "Index settings..." that would be correct.

Reply:
Agree with Jeff, we should standardize our link text. the general standard practice is to have it match section or page title.

suggestion:
"The clone index API operation creates a new target index so you can specify any index settings and aliases to apply to the target index. To learn more, see [Index settings](url) and [Index aliases](url)."